WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentsEditFragment.kt

```diff
@@ -0,0 +1,211 @@
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.snackbar.Snackbar
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.UnifiedCommentsEditFragmentBinding
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.EditCommentActionEvent.CLOSE
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.EditCommentActionEvent.DONE
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.FieldType.COMMENT
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.FieldType.USER_EMAIL
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.FieldType.USER_NAME
+import org.wordpress.android.ui.comments.unified.UnifiedCommentsEditViewModel.FieldType.WEB_ADDRESS
+import org.wordpress.android.ui.pages.SnackbarMessageHolder
+import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.SnackbarItem
+import org.wordpress.android.util.SnackbarItem.Action
+import org.wordpress.android.util.SnackbarItem.Info
+import org.wordpress.android.util.SnackbarSequencer
+import org.wordpress.android.viewmodel.observeEvent
+import javax.inject.Inject
+
+class UnifiedCommentsEditFragment : Fragment(R.layout.unified_comments_edit_fragment) {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    @Inject lateinit var uiHelpers: UiHelpers
+    @Inject lateinit var snackbarSequencer: SnackbarSequencer
+
+    private lateinit var viewModel: UnifiedCommentsEditViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        (requireActivity().application as WordPress).component().inject(this)
+        viewModel = ViewModelProvider(this, viewModelFactory).get(UnifiedCommentsEditViewModel::class.java)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val site = requireArguments().getSerializable(WordPress.SITE) as SiteModel
+        val commentId = requireArguments().getInt(KEY_COMMENT_ID)
+
+        UnifiedCommentsEditFragmentBinding.bind(view).apply {
+            setupToolbar()
+            setupObservers(site, commentId)
+        }
+    }
+
+    private fun UnifiedCommentsEditFragmentBinding.setupToolbar() {
+        setHasOptionsMenu(true)
+
+        val activity = (requireActivity() as AppCompatActivity)
+        activity.setSupportActionBar(toolbarMain)
+        activity.supportActionBar?.let {
+            it.setHomeButtonEnabled(true)
+            it.setDisplayHomeAsUpEnabled(true)
+            it.setHomeAsUpIndicator(R.drawable.ic_cross_white_24dp)
+        }
+    }
+
+    private fun hideKeyboard() {
+        if (!isAdded || view == null) return
+        activity?.let {
```

Comment:
This is more of a question/suggestion than a change request. Why don't we reuse [`ActivityUtils.hideKeyboard`](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/24420c11ec7b4a2557b1a7eb90225642c6a2688b/WordPressUtils/src/main/java/org/wordpress/android/util/ActivityUtils.java#L17) or [`ActivityUtils.hideKeyboardForced `](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/24420c11ec7b4a2557b1a7eb90225642c6a2688b/WordPressUtils/src/main/java/org/wordpress/android/util/ActivityUtils.java#L30) in this case?

Reply:
Good point @antonis , I'm addressing it in [this](https://github.com/wordpress-mobile/WordPress-Android/pull/15401) PR 🙇